src/software/thunderscope/thunderscope.py

```diff
@@ -1,8 +1,13 @@
 import os
```

Comment:
this file is pretty long, any way to split it up?

Reply:
split it up into thunderscope and thunderscope_main